schematron/v2.2.0/tests/data/HVAC_example1.xml

```diff
@@ -130,15 +131,94 @@
+                                </auc:DeliveryType>
+                                <auc:HeatingSourceID IDref="HeatingSource-A" />
+                                <auc:CoolingSourceID IDref="CoolingSource-A" />
+                                <auc:Capacity>123</auc:Capacity>
+                                <auc:CapacityUnits>gpm</auc:CapacityUnits>
+                                <auc:YearInstalled>2020</auc:YearInstalled>
+                                <auc:Quantity>1</auc:Quantity>
+                                <auc:DeliveryCondition>Good</auc:DeliveryCondition>
+                            </auc:Delivery>
+                            <auc:Delivery ID="Delivery-C">
+                                <auc:DeliveryType>
+                                    <auc:ZoneEquipment>
+                                        <auc:Convection>
+                                            <auc:ConvectionType>Chilled beam</auc:ConvectionType>
+                                        </auc:Convection>
+                                    </auc:ZoneEquipment>
+                                </auc:DeliveryType>
+                                <auc:HeatingSourceID IDref="HeatingSource-A" />
+                                <auc:CoolingSourceID IDref="CoolingSource-A" />
+                                <auc:Capacity>123</auc:Capacity>
+                                <auc:CapacityUnits>gpm</auc:CapacityUnits>
+                                <auc:YearInstalled>2020</auc:YearInstalled>
+                                <auc:Quantity>1</auc:Quantity>
+                                <auc:DeliveryCondition>Good</auc:DeliveryCondition>
+                            </auc:Delivery>
+                            <auc:Delivery ID="Delivery-D">
+                                <auc:DeliveryType>
+                                    <auc:ZoneEquipment>
+                                        <auc:Radiant>
+                                            <auc:RadiantType>Radiator</auc:RadiantType>
+                                        </auc:Radiant>
+                                    </auc:ZoneEquipment>
+                                </auc:DeliveryType>
+                                <auc:HeatingSourceID IDref="HeatingSource-A" />
+                                <auc:CoolingSourceID IDref="CoolingSource-A" />
+                                <auc:Capacity>123</auc:Capacity>
+                                <auc:CapacityUnits>gpm</auc:CapacityUnits>
+                                <auc:YearInstalled>2020</auc:YearInstalled>
+                                <auc:Quantity>1</auc:Quantity>
                                 <auc:DeliveryCondition>Good</auc:DeliveryCondition>
                             </auc:Delivery>
                         </auc:Deliveries>
                     </auc:HeatingAndCoolingSystems>
+                    <auc:DuctSystems>
+                        <auc:DuctSystem ID="DuctSystem-A">
```

Comment:
@nllong does requiring declaration of a `auc:DuctSystem` only when we have an `auc:Delivery/auc:DeliveryType/auc:CentralAirDistribution/auc:AirDeliveryType='Central fan'` make sense?

Reply:
no, i wouldn't make this a required existence.

Reply:
Changed to make this not a requirement: 7b064b5